Add 3ds assets, move assets from ctr to pkg/ctr

meepingsnesroms opened the pull request:
NOTE: I do not have the 3ds version of devkitARM installed so I could
not test it if moving the assets to pkg/ctr broke compiling but I did
change the assets paths in Makefile.ctr and Makefile.ctr.salimander so
it should compile.

EDIT: I have verified it compiles.

This should also fix the id clash between fuse, 81 and np2.

meepingsnesroms commented:
I have fixed a few bugs and have tested it on a 3ds, its compiling and working fine.

twinaphex commented:
OK, if you say you have tested it, then it's fine.

Just one thing though - I feel more comfortable if you wait next time to let me merge something. I can look past it for this time but please don't pull the gun like this in future.

meepingsnesroms commented:
I will never self merge an untested pull request.

I normally would not just merge to the main repo, but this bug made 3 cores uninstallable on 3ds and I fixed the assets path simply because I noticed it was inconsistent with other platforms.

I can't just willingly leave something that broken when I have fixed it.

For other changes I will wait though.
